src/styles/index.css

```diff
@@ -0,0 +1,154 @@
+.body {
+  margin: 0;
+}
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
+  color: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+}
+
+.button:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
+  color: #00acdc;
+}
+
+.card {
+  width: 199px;
+  height: 100%;
+  border: 1px solid rgb(240, 7, 228);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.card__image {
+  box-sizing: border-box;
+  background-image: url(/src/images/imac.jpeg);
+  background-size: cover;
+  width: 160px;
+  height: 134px;
+  margin-top: 33px;
+  margin-left: 20px;
+}
+
+.card__name {
+  width: 166px;
+  height: 36px;
+  padding: 40px 16px 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+}
+
+.card__code {
+  width: 98px;
+  height: 14px;
+  padding: 4px 16px;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  color: #616070;
+}
+
+.card:hover {
+  border: 1px solid blue;
+  border-radius: 5px;
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 24px 16px 0;
+}
+
+.card__price-key {
+  width: 31px;
+  height: 18px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  text-align: left;
+  color: #616070;
+}
+
+.card__price-value {
+  width: 50px;
+  height: 18px;
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  text-align: right;
+  color: #060b35;
+}
+
+.review {
+  display: flex;
+  flex-direction: row;
+  padding: 16px 16px 0;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.review__reviews {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.stars {
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star-active {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star-active.svg);
+  box-sizing: border-box;
+  align-items: center;
+  margin: 0 4px 0 0;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  box-sizing: border-box;
+  align-items: center;
+  margin: 0 4px 0 0;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star-active:last-child {
+  margin-right: 0;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

Comment:
It seems that you've directly edited a .css file. In this project, styles should be written in SCSS files, which are then compiled into CSS. Directly editing .css files is not recommended because these changes can be lost when SCSS files are recompiled. Please move your styles to the appropriate SCSS files within the 'src/styles/blocks' directory.